src/builder.rs

```diff
@@ -37,25 +37,25 @@ impl Builder {
 
                 *self.token_frequencies
                      .entry(field_ref.clone())
-                     .or_insert(HashMap::new())
+                     .or_insert_with(HashMap::new)
```

Comment:
Is this like a method reference in Java? Either way, its much cleaner.

Reply:
More or less. It takes object that implements [FnOnce trait](https://doc.rust-lang.org/std/ops/trait.FnOnce.html). This roughly correspond to anything callable (think c++ functors), some of the thinks that implement it are function pointer (this case) or closures.